setup.py

```diff
@@ -5,7 +5,7 @@
 VERSION = "3.0.0"
 
 setup(
-    name="Radicale_IMAP",
+    name="RadicaleIMAP",
```

Comment:
This is unfortunate, it's because https://pypi.org/project/radicale-imap/ is an old fork squatting on your name.